pose-detection/demos/live_video/src/index.js

```diff
@@ -218,25 +211,28 @@ async function app() {
 
   stats = setupStats();
   const isWebGPU = STATE.backend === 'tfjs-webgpu';
-  const useGpuRenderer = (urlParams.get('gpuRenderer') === 'true') && isWebGPU;
   const importVideo = (urlParams.get('importVideo') === 'true') && isWebGPU;
 
-  camera = await Camera.setupCamera(STATE.camera, useGpuRenderer);
+  camera = await Camera.setup(STATE.camera);
 
   await setBackendAndEnvFlags(STATE.flags, STATE.backend);
 
   detector = await createDetector();
-
+  const canvas = document.getElementById('output');
+  canvas.width = camera.video.width;
+  canvas.height = camera.video.height;
+  useGpuRenderer = (urlParams.get('gpuRenderer') === 'true') && isWebGPU;
   if (useGpuRenderer) {
-    const canvas = document.getElementById('output');
-    gpuRenderer = await RendererWebGPU.setup(canvas, importVideo);
+    renderer = await RendererWebGPU.setup(canvas, importVideo);
```

Comment:
A simple line `renderer = new RendererWebGPU(canvas, importVideo)` will not work. This can also be done: 
```
  if (useGpuRenderer) {
    const [device, swapChain] = await getDevice(canvas);
    renderer = new RendererWebGPU(device, swapChain, importVideo);
  } else {
    renderer = new RendererCanvas2d(canvas);
  }
```
Or: 
```
  if (useGpuRenderer) {
    renderer = new RendererWebGPU(canvas, importVideo);
    await render.initialize(); // call getDevice inside.
  } else {
    renderer = new RendererCanvas2d(canvas);
  }
```